pkg/config/config.go

```diff
@@ -87,7 +87,7 @@ func GetConfigFilePath() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	return filepath.Join(path, ".scwrc"), nil
+	return filepath.Join(path, ".scw/config.json"), nil
```

Comment:
Will break the legacy ?

Reply:
It works when `~/.scw` doesn't exist ?

Reply:
Should probably also split the string into `.scw` and `config.json` too, instead of embedding the `/` in it.  [`filepath.Join()`](https://golang.org/pkg/path/filepath/#Join) will add the OS appropriate separator.